tests/specs/nodeRewardsProgram.spec.js

```diff
@@ -0,0 +1,373 @@
+import WalletFixture from '../data/wallet';
+import nem from 'nem-sdk';
+
+const NODE_REWARDS_PROGRAM_API_BASE_URL = 'https://supernodesapi.nem.io';
+
+function mockFetch(returnValue, ok = true) {
+    return spyOn(window, 'fetch').and.returnValue(Promise.resolve({
+        ok,
+        json: () => Promise.resolve(returnValue),
+        text: () => Promise.resolve(returnValue),
+    }));
+}
+
+describe('NodeRewardsProgram service tests', () => {
+    let NodeRewardsProgram;
+    let Wallet;
+
+    beforeEach(angular.mock.module('app'));
+
+    beforeEach(angular.mock.inject((_NodeRewardsProgram_, _Wallet_) => {
+        NodeRewardsProgram = _NodeRewardsProgram_;
+        Wallet = _Wallet_;
+
+        Wallet.use(WalletFixture.mainnetWallet);
+    }));
```

Comment:
Can we put everything in single `beforeEach`?

Reply:
Yes, we can, but it will be inconsistent with other tests in this project. We can update all tests in separate PR if you want.